test/map/map.cpp

```diff
@@ -228,3 +228,39 @@ TEST(Map, RemoveLayer) {
 
     test::checkImage("test/fixtures/map/remove_layer", test::render(map));
 }
+
+TEST(Map, Classes) {
+    MapTest test;
+
+    Map map(test.view, test.fileSource, MapMode::Still);
+    map.setStyleJSON(util::read_file("test/fixtures/api/empty.json"));
+
+    EXPECT_EQ(map.getTransitionOptions().duration->count(), 0);
+
+    auto duration = mbgl::Milliseconds(300);
+    map.setTransitionOptions({ { duration } });
+    EXPECT_EQ(map.getTransitionOptions().duration->count(),
```

Comment:
Can you just do `EXPECT_EQ(map.getTransitionOptions().duration, duration)` here? Ditto line 260.